easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -692,16 +691,14 @@ def det_installversion(version, toolchain_name, toolchain_version, prefix, suffi
 
 
 def fetch_parameter_from_easyconfig_file(path, param):
-    """Fetch parameter specification from given easyconfig file."""
-    # check whether easyblock is specified in easyconfig file
-    # note: we can't rely on value for 'easyblock' in parsed easyconfig, it may be the default value
-    reg = re.compile(r"^\s*%s\s*=\s*(?P<param>\S.*?)\s*$" % param, re.M)
-    txt = read_file(path)
-    res = reg.search(txt)
-    if res:
-        return res.group('param').strip("'\"")
-    else:
-        return None
+    """
+    Fetch parameter specification from given easyconfig file.
+    DEPRECATED: use fetch_parameters_from_easyconfig from easybuild.framework.easyconfigs.parser instead
+    """
+    old = 'fetch_parameter_from_easyconfig_file'
+    new = 'fetch_parameters_from_easyconfig'
+    _log.deprecated("%s is replaced by %s from easybuild.framework.easyconfig.parser" % (old, new), '3.0')
+    return fetch_parameters_from_easyconfig(read_file(path), [param])[0]
```

Comment:
don't read the file in the function call. it's gives messed up stacktraces i think on error